.teamcity/src/main/kotlin/model/FunctionalTestBucketGenerator.kt

```diff
@@ -204,7 +247,7 @@ class FunctionalTestBucketGenerator(private val model: CIBuildModel, testTimeDat
                     it.buildJvmVersion == testCoverage.buildJvmVersion
             }
             foundTestCoverage?.let {
-                buildProjectClassTimes[it.asId("Gradle_Check")]
+                buildProjectClassTimes[it.asId(MASTER_CHECK_CONFIGURATION)]
```

Comment:
I think you need to re-generate the JSON file after this change. Or does it still yield the same result.

Reply:
The JSON is re-generated.